### ISSUE TITLE
Cursor jumps to new note while editing from browser.

### ISSUE THREAD

**Issue by rezzap** [Automattic/simplenote-electron#2085]
### Expected

I would expect that when I'm typing in a note the cursor stays in place and doesn't jump to a new note.

### Observed

Two user's have reported that their cursor jumps to a new note while typing and inserts text into the new note. 

### Reproduced

I have tried to replicate this on a Windows PC and on Mac, but I haven't been able to reproduce it so far. The steps the users' mention are: 

> 1. Go to app.simplenote.com in a Firefox browser (also reported for Chrome but waiting on more details) with all extensions disabled
> 2. Click to create a new note
> 3. See the cursor going back to the previous note and characters inserted there. 

User reports and screenshots/videos available in tickets 2964499-zen and 2935713-zen. 

This seems to happen more regularly when they type in Portuguese or Swedish. I've tried with both French and Swedish keyboards without any problems. 

Make|Model|OS Version|App Version
-|-|-|-
Windows PC||Windows 10|Electron on Firefox 76.0.1

**Comment by msilbers** [comment]
reported in 2938942-zen

Simplenote on web (version 1.16.0-beta2)
ChromeOS (Pixelbook Go, beta channel)

**Comment by msilbers** [comment]
reported in 2957214-zen

Chromebook
Chrome OS 81, using Chrome browser (same version)

"This morning I was in a note I update each day. I was entering, in the body of the note, the time. I entered 7:30 and might have put the space in before going on but after the 7:30 the cursor jumped to the title in a totally different note I have not touched since mid April."

**Comment by msilbers** [comment]
reported in 2955741-zen

Chrome (81.0) iMac with OSX 10.11 (El Capitan)

"The issue appears when I create a new note; while writing on it suddenly the order of the notes change and find myself writing in an older note...
the glitch still persists during writing plain text (maybe accents because I write notes in Spanish and Catalan…)."

**Comment by azollafox** [comment]
Reported in #2964499-zen
User only sees on Firefox 76.0.1 . Does not see in Chrome

**Comment by rachelmcr** [comment]
Thanks for these reports! This looks like a duplicate of https://github.com/Automattic/simplenote-electron/issues/2084. cc @belcherj as we have a sudden influx of these reports and it's very disruptive.

**Comment by belcherj** [comment]
This is my priority. TY!

**Comment by ker8** [comment]
Also happening in Chrome OS Version 81.0.4044.141

For a while now actually I think last few months but cannot tolerate it anymore as its getting in my work. And its unusably slow.

**Comment by gubrist** [comment]
Happening with me with Windows 10, Electron Version 1.16.0.
Jumping to the top of **another, existing note**.

Similar effect as former bug
[https://github.com/Automattic/simplenote-electron/issues/461](url)
which did jump _within_ the actual note.

**Comment by azollafox** [comment]
Another report in Window version 1.16.0 and on Opera 68.0.3618.125
#3028630-zen

**Comment by azollafox** [comment]
Anther report: 
Android app = Phone/Android 8.1 = Simplenote 2.5.1
Web app = Chromebook/Chrome OS 83 = Simplenote 1.16.0-2078

#30149270-zen . User is also reporting sync issues from Android app to the web

**Comment by ker8** [comment]
This problem is getting crazy to be honest and simplenote is no longer usable for months! Plus its gotten very slow lately. I feel sad to leave simplenote though but it left me no choice... I hope their dev team will sort it out soon but its been months now...

**Comment by azollafox** [comment]
Another report at #3029327-zen. Asking for details on what devices/versions of Simplenote they are using

**Comment by gubrist** [comment]
I assume a relation to this "jumping" problem:

https://github.com/Automattic/simplenote-electron/issues/1531

**Comment by belcherj** [comment]
This is resolved on the web. Adding to the 1.18.0 release today which will occur tomorrow. This will not be out on Windows until the certification issues are resolved.
